test/wasm_worker/realloc_wasm_worker.c

```diff
@@ -0,0 +1,81 @@
+#include <emscripten.h>
+#include <emscripten/wasm_worker.h>
+#include <emscripten/em_math.h>
+#include <assert.h>
+#include <string.h>
+
+#define NUM_THREADS emscripten_navigator_hardware_concurrency()
+
+// Stress test realloc() behavior after a suspected instability of realloc() function
+// in https://groups.google.com/g/emscripten-discuss/c/xvcjByZKxVA
```

Comment:
Was there actually some bug effecting realloc or is this test just a part of trying to replicate that issue?   i.e. is this test useful to keep around and run on an ongoing bases?

Reply:
This was intended to try to replicate the issue, but I found no crash with that repro.

Reply:
There didn't exist a test for wasm workers + emmalloc + `realloc()` multithreaded race condition functionality. I think there are several aspects to that which could regress(?)